### ISSUE TITLE
Please Update Your Dependencies Due To PeerDependency Issues

### ISSUE THREAD

**Issue by ajfritz** [balderdashy/sails#2650]
Please update your dependencies. Especially **grunt-contrib-less** solves some **PeerDependency** issues, which causes my Sails npm install to fail as well.

```
npm ERR! peerinvalid The package grunt does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer grunt-contrib-clean@0.5.0 wants grunt@~0.4.0
npm ERR! peerinvalid Peer grunt-contrib-coffee@0.10.1 wants grunt@~0.4.0
npm ERR! peerinvalid Peer grunt-contrib-concat@0.3.0 wants grunt@~0.4.0
npm ERR! peerinvalid Peer grunt-contrib-copy@0.5.0 wants grunt@~0.4.0
npm ERR! peerinvalid Peer grunt-contrib-cssmin@0.9.0 wants grunt@~0.4.1
npm ERR! peerinvalid Peer grunt-contrib-jst@0.6.0 wants grunt@~0.4.0
npm ERR! peerinvalid Peer grunt-contrib-less@0.11.1 wants grunt@^0.4.0
npm ERR! peerinvalid Peer grunt-contrib-uglify@0.4.1 wants grunt@~0.4.0
npm ERR! peerinvalid Peer grunt-contrib-watch@0.5.3 wants grunt@~0.4.0
```

**Comment by loicsaintroch** [comment]
Is this still happening with the latest version of Sails?

**Comment by CWyrtzen** [comment]
Is this still an issue?

**Comment by sailsbot** [comment]
Thanks for posting, @ajfritz.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
